lib/iris/coords.py

```diff
@@ -1233,6 +1233,11 @@ def _guess_bounds(self, bound_position=0.5):
             - the units are degrees,
             - all the points are in the range [-90, 90].
 
+        .. deprecated:: 2.0.0
```

Comment:
You need one blank line after this directive, as per the usage of other directives in this docstring (and elsewhere).